metrics/interface/src/main/java/org/apache/iotdb/metrics/metricsets/cpu/CpuUsageMetrics.java

```diff
@@ -211,6 +217,11 @@ private void updateCpuUsage() {
     lastThreadCpuTime.putAll(currentThreadCpuTime);
     lastThreadUserTime.clear();
     lastThreadUserTime.putAll(currentThreadUserTime);
+    long timeCost = System.nanoTime() - startTime;
+    updateCount.incrementAndGet();
+    if (updateCount.get() % 10 == 0) {
+      log.info("Time for update cpu usage is {} ns", timeCost);
```

Comment:
I recommend changing it to debug logs. If we need to check the time, we can adjust the logback, otherwise the log is too much